src/v/redpanda/application.cc

```diff
@@ -210,17 +210,17 @@ int application::run(int ac, char** av) {
     }
     // use endl for explicit flushing
     std::cout << community_msg << std::endl;
-    vlog(_log.info, "Redpanda {}", redpanda_version());
-    struct ::utsname buf;
-    ::uname(&buf);
-    vlog(
-      _log.info,
-      "kernel={}, nodename={}, machine={}",
-      buf.release,
-      buf.nodename,
-      buf.machine);
 
     return app.run(ac, av, [this, &app] {
+        vlog(_log.info, "Redpanda {}", redpanda_version());
+        struct ::utsname buf;
+        ::uname(&buf);
```

Comment:
presumably this doesn't block we can see several async contexts in seastar in which uname system call is made.

Reply:
that is, can we assume this is not gonna be a problem?

Reply:
yeh this should be fine